pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -92,6 +100,7 @@ type reconciler struct {
 	actualStateOfWorld        cache.ActualStateOfWorld
 	attacherDetacher          operationexecutor.OperationExecutor
 	nodeStatusUpdater         statusupdater.NodeStatusUpdater
+	nodeLister                corelisters.NodeLister
```

Comment:
Had to add a nodeLister to get the node object. Open to suggestions.

Reply:
Seems reasonable to me.  Is there code that initializes this nodeLister?